app/frontend/main.ts

```diff
@@ -16,3 +21,13 @@ const app = new MainWindow({
 });
 
 export default app;
+
+function loadWindowSettings() {
+  let windowSize = getLocalStorage("window.size", [window.outerWidth, window.outerHeight]).value;
+  let windowPosition = getLocalStorage("window.position", [window.screenX, window.screenY]).value;
+  assert(windowSize.length == 2 && windowSize.every(i => sanitize.integer(i)), gt`Bad window size` + windowSize);
+  assert(windowPosition.length == 2 && windowPosition.every(i => sanitize.integer(i)),  gt`Bad window position` + windowPosition);
```

Comment:
This complains if your window is positioned at `[0, 0]`; perhaps `typeof i == "number"` would work better?